data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg

```diff
@@ -40,33 +40,198 @@
         name= _ "Commander Aethyr"
         profile="portraits/aethyr-first.webp"
         canrecruit=yes
-        recruit=Peasant, Bowman, Horseman, Mage, Spearman
-        {GOLD 240 180 120}
-        {INCOME 6 4 2}
+        recruit=Peasant,Bowman,Spearman
+        gold=50 # if you change this, make sure it doesn't mess with the "more expensive recruits" logic
+        income=8
         [ai]
-            aggression=-0.5
             {NO_SCOUTS}
-            recruitment_pattern=mixed fighter,fighter,mixed fighter,archer,fighter,fighter
-            passive_leader=yes
-            grouping=defensive
-        [/ai]
-        [ai]
-            time_of_day=morning,afternoon
-            grouping=offensive
-            aggression=0.5
         [/ai]
         team_name=Haldric
         user_team_name=_"Humans"
         {FLAG_VARIANT loyalist}
     [/side]
-
+    [side]
+        side,color=3,blue
+        no_leader,hidden=yes,yes
```

Comment:
Just checking, but are these okay? They don't seem as bad as the one in `[event]`, at least.

Reply:
I think they should go on separate lines, IMO